src/TestEnvironment.Docker.Containers.Mongo/MongoContainer.cs

```diff
@@ -6,9 +6,6 @@ namespace TestEnvironment.Docker.Containers.Mongo
 {
     public class MongoContainer : Container
     {
-        private readonly string _userName;
```

Comment:
Why do you need changes in this container? You are not inheriting it anywhere

Reply:
Originally, `MongoSingleReplicaSetContainer` was inherited from `MongoContainer`, but then I've realized that it is too complicated for MongoContainer. I've reverted changes in `MongoContainer`.